.gitlab-ci.yml

```diff
@@ -1,6 +1,25 @@
+stages:
+  - test
 
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-commits.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-license.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-python3-format.yml'
+
+test:static:
+  extends: .test
+  stage: test
+  image: python:3.6-buster
+  before_script:
+    - pip install isort
+    - pip install mypy
+    - pip install pylint
+    - pip install .
```

Comment:
:thinking: 
Maybe install the requirements before the app itself?

Reply:
Well, I want to test that `setup.py` has the required requirements there, so that installing the app install also the requirements. Maybe I should remove `requirements.txt` file, as it is unused (I don't think we will add dependabot to this repo, and if so we should modify `setup.py` to read the reqs from the txt)

Reply:
I'll remove the requirements.txt